scl/elasticsearch/elastic-datastream.conf

```diff
@@ -0,0 +1,50 @@
+#############################################################################
+# Copyright (c) 2024 Axoflow
+# Copyright (c) 2024 <szilard.parrag@axoflow.com>
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+@requires json-plugin
+
+block destination elasticsearch-datastream(
+  url()
+  workers(4)
+  batch_lines(100)
+  timeout(10)
+  record("--scope rfc5424 --exclude DATE --key ISODATE @timestamp=${ISODATE}")
+  headers("Content-Type: application/x-ndjson")
+  body_suffix("\n")
+  ...)
+{
+
+@requires http "The elasticsearch-datastream() driver depends on the AxoSyslog http module, please install the axosyslog-mod-http (Debian & derivatives) or the axosyslog-http (RHEL & co) package"
```

Comment:
I get it that this is a backport, but it still depends on the syslog-ng http destination.